packages/web-frontend/src/screens/desktop/RootScreen/index.js

```diff
@@ -24,15 +24,29 @@ import SidePanel from '../../../containers/SidePanel';
 import { EnlargedDialog } from '../../../containers/EnlargedDialog';
 import SessionExpiredDialog from '../../../containers/SessionExpiredDialog';
 import OverlayDiv from '../../../containers/OverlayDiv';
-import { OverlayContainer } from '../../../utils';
-import { TOP_BAR_HEIGHT } from '../../../styles';
+import { DIALOG_Z_INDEX, TOP_BAR_HEIGHT } from '../../../styles';
 import './desktop-styles.css';
 
+const OverlayContainer = styled.div`
+  position: fixed;
+  z-index: ${DIALOG_Z_INDEX};
+  flex-direction: column;
+  flex-wrap: nowrap;
+  width: 100%;
+  height: 100%;
+  pointer-events: visiblePainted; /* IE 9-10 doesn't have auto */
+  pointer-events: none;
+  display: flex; /* Took me ages to find this, is the magic touch */
+  align-items: stretch;
+  align-content: stretch;
+`;
+
 export const RootScreen = ({ enlargedDialogIsVisible, isLoading }) => {
   return (
     <div>
       {/* The order here matters, Map must be added to the DOM body after FlexContainer */}
       <OverlayContainer>
+        <EnvBanner />
```

Comment:
Move the EnvBanner inside the fixed position container so that the total height is not more than 100%